modules/ROOT/pages/manage-exchange-api-task.adoc

```diff
@@ -44,13 +52,49 @@ For example, if an API is defined with the `Implementation URI` set to `http://j
 |`/` | `https://domain.cloudhub.io/users` | `http://jsonplaceholder.typicode.com/users`
 | `/api` | `https://domain.cloudhub.io/api/users` | `http://jsonplaceholder.typicode.com/users`
 |===
-. Optionally, expand the *Advanced options* section and specify additional details:
-* Specify the validations and an API instance label for the API.
-* If you are creating an endpoint with proxy, also specify a scheme (HTTP or HTTPS) and a port, or accept the defaults.
-* Specify a response timeout, if required. 
-* If you choose *Hybrid* as the proxy deployment target, select the *Reference user domain* checkbox to use an API gateway domain.
-+
-You must install the API gateway domain in Mule 3.8 and later.
-+
-. Click *Save*.
+. Optionally, to fine tune additional configurations for the API, expand *Advanced options* and specify the <<field-reference,details>>, and then click *Save*.
+
+[[field-reference]]
+== Field Input Reference
+
+The input that you are required to provide when adding an API from Exchange differs on several factors, such as whether you want to create an endpoint with proxy, and whether the API was created using Mule 3 or Mule 4:
+
+[%header%autowidth.spread,cols="15%,35%,15%,35%"]
+|===
+| Field Name | Description | Required | Notes
+| *Implementation URI* | The URL to access for the proxy or the API. | Yes|  For example, you can use the URL of your API asset in Exchange. In this case, the API is consuming the JSONPlaceholder API, so enter `+http://jsonplaceholder.typicode.com+`.
+| *Proxy version* | Specifies the version of the proxy to use for the endpoint.| No |
+| *Enable Console* | Specifies whether you can simulate your API specification. | No | You can specify a different path in *Console Path*, for example, "/spec/*".
+| *Validations* | Specifies whether to validate inbound requests against a provided specification. | No | You can configure the validations to verify incoming request against provided specifications or by using query parameters.
+| *Scheme* | Specifies whether to use HTTP or HTTPS for the validations. | Yes | If you select HTTPS, specify a TLS context for inbound traffic.
+| *API instance label* | Specifies a label for the API. | No | If you have multiple managed instances of the same API, add a label to differentiate each instance from the another.
+| *Port* | Specifies the number to use if the displayed port is incorrect. | No | 
+| *Response timeout* | Specifies the duration after which a request times out. | No | 
+| *Reference user domain* | Specifies whether to use an API gateway domain. | No | If you chose *Hybrid* as the proxy deployment target previously in the configuration, ensure that you select this option. You must install the API gateway domain in Mule 3.8 and later.
+|===
+
+
+== See Also
+
+* xref:apikit::apikit-simulate[Test APIs with API Console]
+
+// +
+// If you select HTTPS you can optionally specify a TLS context for inbound traffic.
```

Comment:
Placement of == See also?

Reply:
What are the following lists and steps supposed to follow?

Reply:
Commented out. Please ignore.